applications/community-health-check.md

```diff
@@ -0,0 +1,291 @@
+
+Early development of this project has been funded by [Aragon](https://forum.aragon.org/t/real-time-community-health-analytics), Polygon DAO, [Aave](https://noturhandle.gitbook.io/rndao/proposals/proposals/grant-proposal-aave), [MetaCartel](https://forum.metacartel.org/t/grant-request-community-health-a-project-by-rndao/2354) and Near.
+
+## Development Roadmap :nut_and_bolt:
+
+
+### Overview
+
+- **Total Estimated Duration:**  10 weeks
+- **Full-Time Equivalent (FTE):**  6
+- **Total Costs:**  15 000 USDC
+
+We are following a grant-matching processes to spread the risk among different communities. 
+
+### Milestone 1 - Defining metrics
+
+- **Estimated duration:** 4 weeks
+- **FTE:**  1
+- **Costs:** 5000 USDC
+
+Research social network metrics that are viable to measure the health of a community on Twitter, and how they integrate with our existing community health framework. This will build on our existing framework, extending it to metrics that have been tested and used in Twitter communities. For example, we will look into approaches to build the network, virality and clustering/fragmentation of very large online communities. 
+
+Note: We have done an evaluation to decide wheter Reddit or Twitter would be a better option. In the discussion we had with other communities, we were more often asked about an integration with Twitter than with Reddit. Hence, from a scaling perspective, we decided that focusing first on Twitter makes more sense with us.
+
+That being said, we realize that Twitter is undergoing a lot of changes, and we might have to do a last minute pivot to another platform. We are building our analytical scripts in such a way that they can easily be used for other platforms. Therefore, the only thing that has to be changed in the milestones is the name of the platform. 
+
+
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| 0a. | License | The results will be published open-access (no paywall or copyright). |
+| 0b. | Documentation | No documentation necessary.|
+| 0c. | Testing and Testing Guide | Testing and Testing Guide not necessary. |
+| 0d. | Docker | A docker file is not necessary for using the community health check. |
+| 0e. | Article| We will publish an article (technical document) describing the metrics, the insights (so-what), and limitations. This article will also explain how the Twitter community is build (e.g., who are nodes, when there is an edge between two people, who is excluded/included and why). The article will not be behind a paywall. The article will be written for an audience comfortable with data analysis. |
```

Comment:
```suggestion

Milestone 1 will be focused on research. Hence, we're not going to deliver the following usually mandatory deliverables:

- Documentation
- Testing and Testing Guide
- Docker

| Number | Deliverable | Specification |
| -----: | ----------- | ------------- |
| 0a. | License | The results will be published open-access (no paywall or copyright). |
| 0b. | Article | We will publish an article (technical document) describing the metrics, insights (so-what), and limitations. This article will also explain how the Twitter community is built (e.g., who are nodes, when there is an edge between two people, who is excluded/included, and why). The article will not be behind a paywall. The article will be written for an audience comfortable with data analysis. |
```

Reply:
I still think it's easier to skip the deliverables that you're not delivering. This way, it will also be easier to validate the results.